<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-75OLIUH" name="MC.TestSetF" tests="13" failures="1" timestamp="20 Aug 2018 04:24:23 GMT" time="903.147" errors="0">
  <testcase name="TC_10_MC_ME_C" time="63.336" classname="MC.TestSetF"/>
  <testcase name="TC_404_MC_GC_C" time="59.938" classname="MC.TestSetF"/>
  <testcase name="TC_402_MC_GC_A" time="65.688" classname="MC.TestSetF"/>
  <testcase name="TC_412_DownAndUPonly_FF" time="97.359" classname="MC.TestSetF"/>
  <testcase name="TC_403_MC_GC_B" time="67.435" classname="MC.TestSetF"/>
  <testcase name="TC_409_MC_ME_B" time="42.686" classname="MC.TestSetF"/>
  <testcase name="TC_405_MC_FF_A" time="107.511" classname="MC.TestSetF"/>
  <testcase name="TC_408_MC_ME_A" time="44.699" classname="MC.TestSetF"/>
  <testcase name="TC_413_DownAndUPonly_ME" time="59.148" classname="MC.TestSetF"/>
  <testcase name="TC_411_DownAndUPonly_CHROME" time="72.686" classname="MC.TestSetF"/>
  <testcase name="TC_407_MC_FF_C" time="77.256" classname="MC.TestSetF"/>
  <testcase name="CalculateMortgatePayment_HardCodedValues" time="68.233" classname="MC.TestSetF">
    <failure type="java.lang.AssertionError" message="expected [$2,602.75] but found [$2,526.41]">
      <![CDATA[java.lang.AssertionError: expected [$2,602.75] but found [$2,526.41]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at MC.TestSetF.CalculateMortgatePayment_HardCodedValues(TestSetF.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- CalculateMortgatePayment_HardCodedValues -->
  <testcase name="TC_406_MC_FF_B" time="77.172" classname="MC.TestSetF"/>
</testsuite> <!-- MC.TestSetF -->
